docker/dev/docker.env

```diff
@@ -35,3 +35,5 @@ SERVICE_DNS=localhost
 
 GF_SECURITY_ADMIN_USER=admin    
 GF_SECURITY_ADMIN_PASSWORD=password
+
+prometheus_multiproc_dir=/code/monitoring/prometheus/django
```

Comment:
@burnerlee what is this for?

Reply:
@Ram81 for aggregating the metrics of the 4 workers running for our django instance.. there were 2 options: 
1. to create a separate route for each worker and scrape metrics from each of them but it won't be really good.
2. The second was to aggregate metrics on the application level using a common registry for all the workers, which is maintained by django-prometheus by itself. For this it creates some temporary files in this directory and we have to set the environment variable for this. this env variable has to be set in lowercase only so we won't be able to make it uppercase.. however i have made a PR on this library to add support for upper case as well

Reply:
@burnerlee have you checked how much data it stores? We don't want disk to become a bottleneck for metrics